### DISCUSSION TITLE
"Invalid Refresh Token" error.

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#7511] [Questions]
Hi! I'm building an app with Next.js and Supabase. I'm new to this. From time to time I'm getting an "Invalid Refresh Token" error. I'm not sure why.

Can you help me figure out how to fix it?

**Comment by GaryAustin1** [comment]
Usually this is caused by you signing out on another device/tab, or possibly using the same refresh token twice with multiple tabs or code error.  When the jwt is about to expire, expired, the refresh token is used to refresh the jwt.  The refresh does on expire with out being used or canceled.

**Reply by lumenwrites** [reply]
Thank you for your reply! I'm not sure why, but I keep getting this error, even in production.

Can you help me find a way to debug or handle it correctly?

**Comment by jeantje** [comment]
Did you find a solution for this problem??
